app/models/katello/pool.rb

```diff
@@ -100,8 +110,12 @@ def default_sort
       Pool.joins(:subscription).order("subscription.name")
     end
 
-    class Jail < ::Safemode::Jail
-      allow :id, :name, :available, :quantity, :product_id, :contract_number, :type, :account_number, :start_date, :end_date, :organization, :consumed
+    class Pool::Jail < ::Safemode::Jail
```

Comment:
was this change necessary or intentional? it seems weird because this class is basically called 'Katello::Pool::Pool::Jail` which is a little redundant

Reply:
It was to be consistent with how the jail is defined in `host_managed_extensions.rb` but I didn't realize this would result in this weird naming.  Changing it back.